examples/projections/cyl/cyl_oblique_mercator_1.py

```diff
@@ -0,0 +1,27 @@
+"""
+Oblique Mercator
+================
+
+Oblique configurations of the cylinder give rise to the oblique Mercator
+projection. It is particularly useful when mapping regions of large lateral
+extent in an oblique direction. Both parallels and meridians are complex
+curves. The projection was developed in the early 1900s by several workers.
+
+**oa**\ *lon0/lat0* */azimuth*\ */scale*\ **[+v]** or
+**OA**\ *lon0/lat0* */azimuth*\ */width*\ **[+v]** or
+
+The projection is set with **o** or **O**. The central meridian is set by
+*lon0/lat0*. The oblique equator is set by the azimuth. Align the y-axis
+with the optional *+v*. The figure size is set with *scale* or *width*.
+"""
+import pygmt
+
+fig = pygmt.Figure()
+# Using the origin and azimuth
+fig.coast(projection="Oa-120/25/-30/12c+v",
+    region= "-122/35/-107/22+r",
+    frame="afg",
+    land="gray",
+    water="lightblue",
```

Comment:
```suggestion
    shorelines="1/thin",
    water="lightblue",
```

Personal preference, but I think this makes the figure's coast more apparent.